### DISCUSSION TITLE
Radeon ProRender 2.4.6 for Houdini

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD#667] [Announcements]
Radeon ProRender 2.4.6 for Houdini, enhances both viewport and final rendering modes:

- RPR Final uses HIP instead of OpenCL.
- RPR Interactive (using Vulkan and hardware ray tracing) now includes RESTIR-based lighting calculations and FSR upsampling for faster interactive viewport rendering.

Plus many more changes.  See the [changelog](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/master/CHANGELOG.MD) for more details.

<hr /><em>This discussion was created from the release <a href='https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/releases/tag/v2.4.6'>Radeon ProRender 2.4.6 for Houdini</a>.</em>

**Comment by Another52** [comment]
How do I add outlines? In blender it seems to easy although I have never used that. Contour Mode does nothing. The toon shader is very nice but having outlines is necessary
